src-docs/src/views/app_context.js

```diff
@@ -44,6 +47,7 @@ export const AppContext = ({ children }) => {
   };
 
   const isLocalDev = window.location.host.includes('803');
+  setEuiDevProviderWarning(isLocalDev ? 'error' : 'warning'); // Note: this can't be in a useEffect, otherwise it fires too late for style memoization warnings to error on page reload
```

Comment:
I think this should be this to work for warnings:

```suggestion
  setEuiDevProviderWarning(isLocalDev ? 'error' : 'warn'); // Note: this can't be in a useEffect, otherwise it fires too late for style memoization warnings to error on page reload
```

Reply:
Argh! Thanks so much for catching that!! 🙈